packages/seller-client/src/components/ProductListItem/ProductListItem.tsx

```diff
@@ -58,22 +59,48 @@ const ProductListItem: React.FC<ProductListItemProps> = ({
             <Price price={price} />
           </Typography>
         </Box>
-        <Box height={24} display="flex" alignItems="center">
-          <Typography
-            variant="caption"
-            style={{ color: getColorForStatus(theme, status) }}
-          >
-            {status.toUpperCase()}
-          </Typography>
-          <Spacer width={16} />
-          <Typography variant="caption">
-            <strong>{optionsCount} Options</strong>
-          </Typography>
-          <Spacer width={8} />
-          <Typography variant="caption">
-            <em>{groupNames?.join(', ') ?? 'Individual Options'}</em>
-          </Typography>
+        <Box display="flex" alignItems="center">
+          {status && (
+            <>
+              <Typography
+                variant="caption"
+                style={{ color: getColorForStatus(theme, status) }}
+              >
+                {status.toUpperCase()}
+              </Typography>
+              <Spacer width={16} />
+            </>
+          )}
+          {optionsCount && optionsCount > 2 && (
+            <>
+              <Typography variant="caption">
+                <strong>{optionsCount} Options</strong>
+              </Typography>
+              <Spacer width={16} />
+            </>
+          )}
+          {groupNames != null && (
```

Comment:
JavaScript with the `!==` vs. `!=`. I think this actually works but it should really be `groupNames !== undefined` or

```suggestion
          {groupNames && (
```

Reply:
Yea `!= null` would cover the cases where it's `null` or `undefined` but you're right in the sense that `groupnames` will never be `null`. Updated the check in cff2ec8c7d46a49eff5c08106e62ea67ab17e2fe